src/elements/content-sidebar/BoxAISidebar.scss

```diff
@@ -4,60 +4,52 @@
 
```

Comment:
Hi Kamil,
A clarification question:
1. Instead of defining media query here you leverage media query created for the whole sidebar ([here](https://github.com/box/box-ui-elements/blob/78e4026aaa5bf6d617685ba402511e787c182809/src/elements/content-sidebar/SidebarContent.scss)) by appending the bcs-is-wider class?

Reply:
Not really. Media query block was not needed after a small refactor I've done by not setting the Box AI Sidebar width to a fixed number (`$sidebarContentIncreasedWidth`). It's now using the entire available width in both scenarios.

And the addition `.bcs-is-wider` was required to get a higher specificity for some of the selectors. We're safe to add it because `bcs-is-wider` will always be present when  Box AI Sidebar is visible.